### PR TITLE
feat(stresschaos): support cgroup v2 for docker and cri-o

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <im@strrl.dev>

### What problem does this PR solve?

close https://github.com/chaos-mesh/chaos-mesh/issues/2978
close https://github.com/chaos-mesh/chaos-mesh/issues/2069

### Related changes

- [ ] This change also requires further updates to the [website](https://github.com/chaos-mesh/website) (e.g. docs)
- [ ] This change also requires further updates to the `UI interface`
- Need to **cheery-pick to release branches**
  - [x] release-2.4
  - [x] release-2.3

### Checklist

CHANGELOG

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [x] Manual test (add steps below)

Side effects

- [ ] **Breaking backward compatibility**

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
I have manually tested this PR on all supported CRI: docker, contained, crio, and it just works...

**Comment by Andrewmatilde** [comment]
please execute make check on your local machine :)

**Comment by STRRL** [comment]
> please execute make check on your local machine :)

Updated, PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 ### Changed
 
 - Use the next generation `New Workflow` UI by default [#3718](https://github.com/chaos-mesh/chaos-mesh/pull/3718)
+- Support cgroup v2 for docker and crio [#3698](https://github.com/chaos-mesh/chaos-mesh/pull/3698)
```
Comment (g1eny0ung) [inline CHANGELOG.md:22]: I think this description should be more detailed. To add it is for StressChaos.

**Review by g1eny0ung — APPROVED** [review]
I've tested it in my local arm64 docker and it works fine.

File: CHANGELOG.md (hunk 1 above)
Comment (STRRL) [reply CHANGELOG.md]: Updated!

**Comment by STRRL** [comment]
/merge


I removed the tag `cherry-pick-release-2.3` because we do not maintain that minor version.

**Pull request merged**